src/views/video/VideoView.integrationTest.tsx

```diff
@@ -101,6 +101,27 @@ describe('Video View', () => {
     expect(await wrapper.findByLabelText('Copy video link')).toBeVisible();
   });
 
+  it('renders learning outcome if user has BO_WEB_APP_DEV flag', async () => {
```

Comment:
Would also test the case where the learning outcomes are not rendered if we get an error back when retrieving them.